readme.md

```diff
@@ -694,6 +694,7 @@
 - [Network Analysis](https://github.com/briatte/awesome-network-analysis#readme)
 - [PCAPTools](https://github.com/caesar0301/awesome-pcaptools#readme)
 - [Real-Time Communications](https://github.com/rtckit/awesome-rtc#readme) - Network protocols for near simultaneous exchange of media and data.
+- [SNMP](https://github.com/eozer/awesome-snmp#readme) - A protocol for collecting and organizing information about managed devices on IP networks and for modifying that information to change device behavior.
```

Comment:
It would be great if you could shorten the description a tiny bit.

Reply:
Agreed. Pushed a new commit.